helpers/calendar/credentials.py

```diff
@@ -34,3 +34,27 @@ def set_api_credentials(self):
         service = build('calendar', 'v3', developerKey=api_key,
                         http=credentials.authorize(Http()))
         return service
+
+
+def get_google_calendar_events(calendarId=None, timeMin=None, timeMax=None,
+                               singleEvents=None, orderBy=None):
+    credentials = Credentials()
+    service = credentials.set_api_credentials()
+    events = service.events().list(calendarId=calendarId, timeMin=timeMin,
+                                   timeMax=timeMax, singleEvents=singleEvents,
+                                   orderBy=orderBy).execute()
+    return events
+
+
+def get_google_calendar_events_without_limits(calendarId=None):
```

Comment:
good work @bevkololi though I'd like to propose that this function be named `get_all_google_calender_events`

Reply:
on it